object.c

```diff
@@ -4565,6 +4565,143 @@ InitVM_Object(void)
+     * - #eval:: Evaluates the Ruby code in the given string.
+     * - #exec:: Executes the given command.
+     * - #syscall:: Calls the operating system function identified by the given number;
+     *              returns the result or raises an exception if the function failed.
+     * - #system:: Executes the given command in a subshell.
+     *
+     * === Exiting
+     *
+     * - #abort:: Aborts the current process after printing the given arguments.
+     * - #at_exit:: Registers the given block as a proc to be executed
+     *              when the program exits.
+     * - #exit:: Exits the current process after calling any registered
+     *           +at_exit+ handlers.
+     * - #exit!:: Exits the current process without calling any registered
+     *            +at_exit+ handlers.
+     *
+     * === Exceptions
+     *
+     * - #catch:: Executes the given block, possibly catching a thrown object.
+     * - #fail:: Raises an exception based on the given arguments.
+     * - #raise:: Raises an exception based on the given arguments.
+     * - #throw:: Transfers control to the end of the active catch block
+     *            that is waiting for the given tag.
+     *
+     * === Strings
+     *
+     * - #format:: Returns the string resulting from applying the given format string
+     *             to any additional arguments.
+     * - #gsub:: Performs global substitution using the given pattern and either
+     *           a replacement string or a block.
+     * - #sprintf:: Returns the string resulting from applying the given format string
+     *              to any additional arguments.
+     * - #sub:: Performs single substitution using the given pattern and either
+     *          a replacement string or a block.
+     *
+     * === \IO
+     *
+     * - #chomp:: Removes the trailing record separator, if any, from <tt>$_</tt>.
+     * - #chop:: Removes the last character, if any, from <tt>$_</tt>.
+     * - #gets:: Returns and assigns to <tt>$_</tt> the next line from the current input.
+     * - #open:: Creates an IO object connected to the given stream, file, or subprocess.
+     * - #p::  Writes the given objects to standard output.
+     * - #pp:: Prints the given objects in pretty form.
+     * - #print:: Prints the given objects to standard output.
+     * - #printf:: Prints the string resulting from applying the given format string
+     *             to any additional arguments.
+     * - #putc:: Equivalent to <tt.$stdout.putc(object)</tt> for the given object.
+     * - #puts:: Equivalent to <tt>$stdout.puts(*objects)</tt> for the given objects.
+     * - #readline:: Equivalent to Kernel.gets, except that +readline+
+     *               raises an exception at end of file.
+     * - #readlines:: Returns an array of the remaining lines from the current input.
+     * - #select:: From arrays of IO streams, returns the first one to become ready.
```

Comment:
`Same as <tt>IO.select</tt>.`

Reply:
I've let this link through, instead of tt.